src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -414,12 +442,22 @@ export function fetchPastAppointments(startDate, endDate, selectedIndex) {
       });
 
       try {
-        const facilityData = await getAdditionalFacilityInfo(
-          getState().appointments.past,
-          featureFacilitiesServiceV2,
-        );
+        let facilityData;
```

Comment:
I think we also want to make these changes in `fetchPendingAppointments`. Same updates as you've made for the other two places, just checking `featureVAOSServiceRequests` instead of `featureVAOSServiceVAAppointments`.

We could leave this for a separate PR, too, though.

Reply:
Yea, I actually removed it because it seemed like the requests would have already been retrieved from the fetchFutureAppointments call. But I see that 'includeRequests' flag is false.

Reply:
Done